--- Imaging-1.1.6.orig/PIL/Image.py     2006-12-03 12:08:22.000000000 +0000
+++ Imaging-1.1.6/PIL/Image.py  2008-10-02 14:51:43.000000000 +0100
@@ -2065,9 +2065,7 @@
     else:
         format = None
         if not command:
-            command = "xv"
-            if title:
-                command = command + " -name \"%s\"" % title
+            command = "eog"

     if image.mode == "I;16":
         # @PIL88 @PIL101

